src/docfx/build/xref/XrefResolver.cs

```diff
@@ -224,9 +224,14 @@ private void ValidateExternalXref(IReadOnlyDictionary<string, InternalXrefSpec[]
         {
             if (!internalXrefMap.ContainsKey(xrefGroup.Key))
             {
-                _errorLog.Add(Errors.Xref.UidNotFound(
-                    xrefGroup.Key, xrefGroup.Select(xref => xref.ReferencedRepositoryUrl).Distinct(), xrefGroup.First().SchemaType) with
-                { Level = _config.IsLearn ? ErrorLevel.Error : ErrorLevel.Warning });
+                foreach (var item in xrefGroup)
+                {
+                    _errorLog.Add(Errors.Xref.UidNotFound(
+                        xrefGroup.Key,
+                        repository: item.ReferencedRepositoryUrl,
+                        item.SchemaType,
+                        item.PropertyPath) with { Level = _config.IsLearn ? ErrorLevel.Error : ErrorLevel.Warning });
```

Comment:
I think the ultimate goal is to change certificate -> learning path reference into warnings. How are you going to achieve this with the `propertyPath` here?

Reply:
we can control that by validation ruleset, for example https://docsmetadatatool.azurewebsites.net/rules/Learn
![image](https://user-images.githubusercontent.com/226552/146106544-7854d3af-a090-46e4-83da-0d61c7c1cdc8.png)

Reply:
👍to explain this! + @shanluo911 @easonyq for awareness